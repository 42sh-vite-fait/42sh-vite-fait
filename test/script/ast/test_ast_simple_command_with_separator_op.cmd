>file1 a <file2;
