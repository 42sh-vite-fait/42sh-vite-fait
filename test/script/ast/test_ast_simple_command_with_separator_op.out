AST:
SIMPLE_COMMAND: a
	redirections#0: IO=1, Operator=GREAT, Word=file1
	redirections#1: IO=0, Operator=LESS, Word=file2
